packages/plugin/src/theia.d.ts

```diff
@@ -7288,6 +7288,19 @@ declare module '@theia/plugin' {
         */
         export function registerFoldingRangeProvider(selector: DocumentSelector, provider: FoldingRangeProvider): Disposable;
 
+        /**
```

Comment:
please apply formatting

Reply:
Updated whitespaces

Reply:
please apply to the document formatting in Theia or VS Code instead of chaning a line, there are similar issues below